tests/test_meta/test_outlier_classifier.py

```diff
@@ -38,6 +38,8 @@ def test_obvious_usecase_quantile(dataset):
     clf_quantile.fit(X, y)
     assert clf_quantile.predict([[10, 10]]) == np.array([1])
     assert clf_quantile.predict([[0, 0]]) == np.array([0])
+    np.testing.assert_array_almost_equal(clf_quantile.predict_proba([[10, 10]]), np.array([[0, 1]]), decimal=4)
+    np.testing.assert_array_almost_equal(clf_quantile.predict_proba([[0, 0]]), np.array([[1, 0]]), decimal=4)
     assert isinstance(clf_quantile.score(X, y), float)
```

Comment:
As far as tests go I'd argue this is fine. I only wonder if it's perhaps good to also add a parameterized for the outlier detection method. We're only testing `GMMOutlierDetector` here. I wouldn't mind adding [isolation forests](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.IsolationForest.html#sklearn.ensemble.IsolationForest) and [one class svm](https://scikit-learn.org/stable/modules/generated/sklearn.svm.OneClassSVM.html#sklearn.svm.OneClassSVM).

Reply:
sure, i'll do that!